internal/sdk/extra_sdk_options.go

```diff
@@ -150,13 +162,74 @@ func NewWithCredentials(ctx context.Context, cred *ClientCredentials, opts ...Cu
 	}
 
 	sdkOpts := []SDKOption{}
-	if options.useWithServer {
-		sdkOpts = append(sdkOpts, WithServerURL(options.ServerURL))
+	if options.UseWithServer() {
+		sdkOpts = append(sdkOpts, WithServerURL(options.ServerURL()))
 	}
-	if options.useWithTenant {
-		sdkOpts = append(sdkOpts, WithTenantDomain(options.Tenant))
+	if options.UseWithTenant() {
+		sdkOpts = append(sdkOpts, WithTenantDomain(options.Tenant()))
 	}
 	sdkOpts = append(sdkOpts, WithClient(uclient))
 
 	return New(sdkOpts...), nil
 }
+
+func NormalizeTenant(input string) (*ClientConfig, error) {
+	input = strings.ToLower(input)
+
+	var err error
+	u := &url.URL{}
+	if !strings.Contains(input, "://") {
```

Comment:
Thanks for adding the colon like in [the go sdk pr](https://github.com/ConductorOne/conductorone-sdk-go/pull/53).